server/manager/src/main/java/org/apache/accumulo/manager/state/MergeStats.java

```diff
@@ -271,6 +281,22 @@ private boolean verifyMergeConsistency(AccumuloClient accumuloClient, CurrentSta
         && unassigned == verify.total;
   }
 
+  @VisibleForTesting
+  boolean verifyState(MergeInfo info) {
+    if (info.getState() != MergeState.WAITING_FOR_OFFLINE) {
+      return false;
+    }
+    return true;
```

Comment:
Another merge state seems really unexpected here, wonder if an exception would be better.

```suggestion
    Preconditions.checkstate(info.getState() == MergeState.WAITING_FOR_OFFLINE, "Unexpected merge state %", info.getState());
    if (info.getState() != MergeState.WAITING_FOR_OFFLINE) {
      return false;
    }
    return true;
```